src/tiled/mapdocument.cpp

```diff
@@ -335,6 +335,21 @@ void MapDocument::resizeMap(const QSize &size, const QPoint &offset, bool remove
     // TODO: Handle layers that don't match the map size correctly
 }
 
+
```

Comment:
I generally only separate methods by one empty line, so there is essentially one empty line too much here.